### PR TITLE
Add iA Presenter

### PR DESCRIPTION
**Pull request by andre4ik3**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/i/ia-presenter.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  homepage "https://ia.net/presenter"
+
+  livecheck do
+    url "https://iapresenter.s3.amazonaws.com/release/updates.xml"
+    strategy :sparkle do |item|
+      "#{item.short_version}-#{item.version}"
+    end
+  end
+
+  depends_on macos: ">= :monterey"
```
Comment (SMillerDev) [inline Casks/i/ia-presenter.rb]: Upstream defines it as:
```suggestion
  depends_on macos: ">= : big_sur"
```

Comment (andre4ik3) [reply Casks/i/ia-presenter.rb]: Yup, already changed it. Not sure if it defines the correct version tho? It needs 11.3, not 11.0, I didn't see a way to put 11.3 in the cask cookbook so I just put monterey.

CleanShot 2024-01-06 at 16 30 09@2x

File: Casks/i/ia-presenter.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+cask "ia-presenter" do
+  version "1.1.2-11002"
+  sha256 "ce026bfc2155104e68e5eeb9eaa29d0a279e7c5de7771ea55e19ff5a4ea2740e"
+
+  url "https://files.ia.net/presenter/release/iA-Presenter-#{version}.zip"
+  name "ia-presenter"
```
Comment (bevanjkay) [inline Casks/i/ia-presenter.rb:6]:
```suggestion
  name "iA Presenter"
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]

File: Casks/i/ia-presenter.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  desc "Create presentation slides from a Markdown document"
+  homepage "https://ia.net/presenter"
+
+  livecheck do
+    url "https://iapresenter.s3.amazonaws.com/release/updates.xml"
+    strategy :sparkle do |item|
+      "#{item.short_version}-#{item.version}"
+    end
```
Comment (bevanjkay) [inline Casks/i/ia-presenter.rb:14]: Should use the default sparkle here that returns the versions separated by a comma. 

Then update the url using `#{version.csv.first}-#{version.csv.second}`

**Review by bevanjkay — CHANGES_REQUESTED** [review]

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @andre4ik3

**Pull request merged**